### PR TITLE
Private/devidask/remove ntp installation

### PR DESCRIPTION
**Pull request by devidask27**

### CODE REVIEW

File: pkg/platform/debian/debian.go [hunk 1]
```diff
@@ -356,3 +358,29 @@ func (d *Debian) checkPIDofSystemd() (bool, error) {
 		return true, nil
 	}
 }
+
+func (d *Debian) checkIfSystemdTimesyncdServiceRunning() {
```
Comment (nikhilbandi) [inline pkg/platform/debian/debian.go:362]: I thought the logic was:
Check if timesyncd is running
  if not, check if ntp is running
    if not, start timesyncd, which id present by default in Ubuntu 16.04 + versions.

This should get rid of checking for any versions.

I am missing something here?

Also, please write this logic in the function in the form of comments.

File: pkg/platform/debian/debian.go [hunk 2]
```diff
@@ -356,3 +358,29 @@ func (d *Debian) checkPIDofSystemd() (bool, error) {
...
+			} else {
+				zap.S().Debug("installing systemd-timesyncd")
+				d.installOSPackages("systemd-timesyncd")
+				zap.S().Debug("Starting systemd-timesyncd service")
+				if _, err := d.exec.RunWithStdout("bash", "-c", "systemctl start systemd-timesyncd"); err != nil {
+					zap.S().Fatalf("Failed to start systemd-timesyncd")
+				}
+			}
+		} else {
+			zap.S().Debug("NTP service is running. skiping the installation.")
```
Comment (nikhilbandi) [inline pkg/platform/debian/debian.go:381]: skiping => Skipping

File: pkg/platform/debian/debian.go [hunk 3]
```diff
@@ -356,3 +358,29 @@ func (d *Debian) checkPIDofSystemd() (bool, error) {
...
+		}
+	} else {
+		zap.S().Debug("systemd-timesyncd service is running. Skiping the installation.")
```
Comment (nikhilbandi) [inline pkg/platform/debian/debian.go:384]: Skiping => Skipping

File: pkg/pmk/node.go [hunk 4]
```diff
@@ -223,10 +223,10 @@ func installHostAgentCertless(ctx Config, regionURL string, auth keystone.Keysto
 	}
 
 	if IsRemoteExecutor {
-		cmd = fmt.Sprintf(`bash /tmp/installer.sh --no-proxy --skip-os-check --ntpd %s`, installOptions)
```
Comment (nikhilbandi) [inline pkg/pmk/node.go:226]: What's the significance of replacing option ntpd with no-ntp?

Comment (devidask27) [reply pkg/pmk/node.go:226]: it will skip ntp installation.

File: pkg/platform/debian/debian.go (hunk 1 above)
Comment (devidask27) [reply pkg/platform/debian/debian.go:362]: Okay i will do the change.

Comment (devidask27) [reply pkg/platform/debian/debian.go]: Done

**Review by anupbarve — APPROVED** [review]

**Pull request merged**